core/src/consensus/tendermint/engine.rs

```diff
@@ -137,12 +137,16 @@ impl ConsensusEngine for Tendermint {
     fn on_close_block(
         &self,
         block: &mut ExecutedBlock,
-        parent_header: &Header,
-        parent_common_params: &CommonParams,
         term_common_params: Option<&CommonParams>,
```

Comment:
Can you remove this parameter too?

Reply:
I tried to do that, and I thought it'd be a bit complex for now.
How about removing this after https://github.com/CodeChain-io/codechain/pull/1895 is merged?